public/style.css

```diff
@@ -1,51 +1,132 @@
-.sidebar {
-    position: fixed;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    z-index: 100;
-    padding: 90px 0 0;
-    box-shadow: inset -1px 0 0 rgba(0, 0, 0, .1);
-    z-index: 99;
+
+
+:root, [data-bs-theme=light] {
+  --bs-breadcrumb-margin-bottom: 0.2rem;
 }
 
-@media (max-width: 767.98px) {
-    .sidebar {
-        top: 11.5rem;
-        padding: 0;
+/* Defaults. */
+/* for extra small screens, show a similar sidebar but full width and only if the menu button is pressed. */
+@media(max-width: 768px) {
```

Comment:
as i said before i want the menu items to always be in the menu.
it currently occupies the space for the title which is primary information.

also the menu had a slide open animation that is missing now

Reply:
I hid the medium options now, but as I said, how about leaving this option open to the user?

The animation was never visible to me (Firefox 125) so I can't check this.

Reply:
![image](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/assets/5467398/fc8b555c-9098-4d12-b6eb-5cd6f61eff6c)
it works for me just fine

Reply:
custom themes can have those buttons in the title bar thats fine but the primary shipped theme will not

Reply:
They are not in the top bar any more. Actually, except for the menu icon and the (just decorative) title "Reports", the top menu bar is now completely without function. We could also remove it completely. What do you think?

Reply:
Not entirely, the burger menu is still there. 
I will postpone this for another PR to get this one merged.